include/RAJA/policy/sycl/reduce.hpp

```diff
@@ -251,7 +251,7 @@ struct TargetReduce
   {
 #ifdef __SYCL_DEVICE_ONLY__
     auto i = 0; //__spirv::initLocalInvocationId<1, cl::sycl::id<1>>()[0];
-    auto atm = cl::sycl::ext::oneapi::atomic_ref<T, cl::sycl::memory_order_acq_rel, cl::sycl::memory_scope::device, cl::sycl::access::address_space::global_space>(val.device[i]);
+    auto atm = ::sycl::atomic_ref<T, cl::sycl::memory_order_acq_rel, cl::sycl::memory_scope::device, cl::sycl::access::address_space::global_space>(val.device[i]);
```

Comment:
Hi @homerdin, I've built a new SYCL compiler from scratch on LLNL's Corona machine (compiler build process here https://github.com/LLNL/RAJA/issues/1255), and ran in to some SYCL API problems that were solved by changing some namespaces. Are these changes ok with you?